source/features/linkify-user-location.tsx

```diff
@@ -26,13 +33,17 @@ const hovercardObserver = new MutationObserver(([mutation]) => {
 
 function init(): void {
 	addLocation(document.body);
+}
 
+function hovercardInit(): void {
 	const hovercardContainer = select('.js-hovercard-content > .Popover-message');
 	if (hovercardContainer) {
 		hovercardObserver.observe(hovercardContainer, {childList: true});
 	}
 }
 
 void features.add(import.meta.url, {
-	init: onetime(init),
+	init,
```

Comment:
I think we should have a `include` here, but where does this feature actually works (excluding the hovercard part)? I can think of user profiles and user followers/following, is there anything else?

Reply:
Sounds right. I think this is specifically for profiles and hovercards.